Felienne/hedy
# [CLEAN-UP] Removes fixed error strings, adds dynamic country selector

**Description**
In this PR we fix two small issues. First we clean a few hard-coded strings that where still placed on the `/verify` route, always returning an English string if something went wrong. Secondly we replace the currently hard-coded countries dictionary by a library based one. Cleaning-up the code and making sure we are up-to-date with possible changes in countries, abbreviations and so on.

**Fixes**
This PR fixes #2287 and fixes #2415

**How to test**
This is a clean-up task, if nothing is broken it works. However, it might be nice to verify that the country selector does indeed still work on the signup page as well as the update profile section.
